### ISSUE TITLE
simple dnn(2 layers) with softmax output with multi cross entropy and adam optimizer is not matching with keras output

### ISSUE THREAD

**Issue by samathasolupati** [tiny-dnn/tiny-dnn#1028]

**Comment by Ravenwater** [comment]
care to elaborate?

**Comment by samathasolupati** [comment]
Iam trying to build a model which having 2hidden layers,softmax and using adam optimizer. Same model i build with keras. there performance is good. on tiny_dnn performance is not matching with keras. and not that much good.

**Comment by Ravenwater** [comment]
when you say it doesn't match, would you be able to share the comparison (classification accuracy between the two models), and how the two systems evolve during learning? Simply stating that they are different doesn't give us anything to help you with.
